src/Windows/Avalonia.Direct2D1/Media/FormattedTextImpl.cs

```diff
@@ -109,7 +109,18 @@ private void ApplySpan(FormattedTextStyleSpan span)
 
         private Size Measure()
         {
-            var metrics = TextLayout.Metrics;
+            DWrite.TextMetrics metrics;
+
+            // SharpDX bug
```

Comment:
What is this bug? Is there a SharpDX issue about it? Could you include a link to the issue, or at least a description of the bug?

Reply:
https://github.com/sharpdx/SharpDX/issues/1034